docs/_config.yml

```diff
@@ -27,7 +27,7 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://dafny-lang.github.io/dafny" # the base hostname & protocol for your site, e.g. http://example.com
 
 # Build settings
-theme: minima
+theme: jekyll-theme-slate
```

Comment:
Can you please post a screenshot of the result of this theme (before/after) in the PR description?

Reply:
Oops - my mistake. An experiment I did not revert. I'll fix it separately.